example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.airbnb.android.react.maps.example">
 
-    <uses-sdk android:minSdkVersion="16" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_COURSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <application
-      android:allowBackup="true"
+      android:allowBackup="false"
```

Comment:
Why this?

Reply:
I don't know - this change was made in RN 0.54 https://github.com/ncuillery/rn-diff/compare/rn-0.53.3...rn-0.54.0

Reply:
Ok, It's not a problem for example

Reply:
That's just some code cleanup from RN team, nothing serious.